src/main.py

```diff
@@ -0,0 +1,197 @@
+# search_query.send_keys('site:linkedin.com/in/ AND "Indian Institute of Technology, Kanpur" AND "Pragya Jain" ')
+# search_query.send_keys(Keys.RETURN)
+
+# linkedin_urls = driver.find_elements_by_xpath('//*[@id="rso"]/div[1]/div/div[1]/div/a')
+# time.sleep(5)
+
+linkedin_urls = ['https://www.linkedin.com/in/pragyajain73/?originalSubdomain=in']
+collegesyearofgrad = []
+colleges = []
+degrees = []
+collegesyearofjoining = []
+collegesduration = []
+
+firstjobtitles = []
+firstjobdurations = []
+firstjobcompanies = []
+firstjobjoiningdates = []
+firstjobleavingdates = []
+
+count = 0
+
+
+def removeFullTime(text):
+    temp = text.split()
+    for t in temp:
+        if t == 'Full-time':
+            if (text.replace('Full-time · ', '')) == None:
+                return text.replace(' · Full-time', '')
+            else:
+                return text.replace('Full-time · ', '')
+
+
+def convertingMonthToNumber(month):
+    datetime_object = datetime.datetime.strptime(month, "%b")
+    return datetime_object.month
+
+
+
+for linkedin_url in linkedin_urls:
+    companies = []
+    jobs = []
+    jobsduration = []
+    joiningdates = []
+    leavingdates = []
+    driver.get('https://www.linkedin.com/in/pragyajain73/?originalSubdomain=in')
+    time.sleep(5)
+    src = driver.page_source
+    soup = BeautifulSoup(src, 'html.parser')
+
+
+    #SCRAPING EXPERIENCE SECTION
+
+    experience = soup.find(lambda tag:tag.name=="section" and "Experience" in tag.text).find('ul')
+    formatcheck = experience.find_all('li', {'class':'artdeco-list__item pvs-list__item--line-separated pvs-list__item--one-column'})
+
+    for f in formatcheck:
+        if (f.find('div', {'class' : 'display-flex flex-column full-width align-self-center'})).find('a') == None:
+            job_titles= f.find_all("span", {'class': 'mr1 t-bold'})
+            for j in job_titles:
+                job = j.find("span", {'class' : 'visually-hidden'}).get_text().strip()
+                jobs.append(job)
+
+            company_names = f.find_all("span", {'class' : 't-14 t-normal'})
+            for c in company_names:
+                company = c.find("span", {'class' : 'visually-hidden'}).get_text().strip()
+                removeFullTime(company)
+                companies.append(company)
+
+            joining_datesANDduration = f.find_all("span", {'class': 't-14 t-normal t-black--light'})[0]
+            dateANDduration = joining_datesANDduration.find("span", {'class' : 'visually-hidden'}).get_text().strip()
+            joiningdate = dateANDduration.split(' - ')[0]
+            leavingdate = (dateANDduration.split(' - ')[1]).split(' · ')[0]
+            jobduration = dateANDduration.split(' · ')[1]
+            joiningdates.append(joiningdate)
+            jobsduration.append(jobduration)
+            leavingdates.append(leavingdate)
+
+        else:
+            job_titles = f.find_all('span', {'class': 'mr1 hoverable-link-text t-bold'})[(len(f.find_all('li'))) -1]
```

Comment:
low LOGAF: there should be a way to find the relevant elements without using brittle queries like these class names.